src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -50,6 +51,19 @@ class DetectedLanguageSuitev4 extends TestBase with DataFrameEquality with TextK
     assert(iso(0).get(0).toString == "en" && iso(0).get(1).toString == "es" &&
       iso(1).get(0).toString == "fr" && iso(1).get(1).toString == "zh")
   }
+
+  test("Language Detection - Assert Confidence Score") {
+    val replies = detector.transform(df)
+      .select("output.result.name","output.result.confidenceScore")
+      .collect()
+
+    val confidence = replies.map(row => row.getList(1))
+    assert(confidence(0).get(0).toString == "0.81")

```

Comment:
as mentioned --> the score here might change with new model versions. I would suggest just checking whether the score is larger than 0.5 for ld/ sentiment/ etc.